client/swagger.gradle

```diff
@@ -1,13 +1,20 @@
+def jerseyVersion = '2.40'
+
 dependencies {
-    // Version controlled by dependency management plugin
-    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
-    implementation 'org.glassfish.jersey.core:jersey-client'
-    implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
-    implementation 'org.glassfish.jersey.media:jersey-media-multipart'
-    implementation 'io.swagger.core.v3:swagger-annotations'
-    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
-    implementation "org.glassfish.jersey.inject:jersey-hk2"
-    implementation "org.glassfish.jersey.connectors:jersey-apache-connector"
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.2'
+    implementation "org.glassfish.jersey.core:jersey-client:$jerseyVersion"
```

Comment:
Is there a reason why using the dependency management plugin doesn't pull in the correct version here? The whole point of the plugin is that you shouldn't need to hardcode versions in the gradle config.

Reply:
The plugin management is pulling versions from Spring Boot 3, but we need to make our client compatible with Spring Boot 2, so we need to manually pull older versions.